src/opening-period/form/OpeningPeriodForm.scss

```diff
@@ -84,7 +85,21 @@
 }
 
 .opening-period-final-action-row-container {
-  margin-top: 116px;
+  height: $spacing-layout-xl;
+  position: fixed;
```

Comment:
Could it actually be a sticky action row?

Reply:
With sticky I have problem of getting it 100% width of the page 🤔